src/PimcoreMonitorBundle/Command/HealthReportCommand.php

```diff
@@ -76,10 +77,10 @@ protected function configure(): void
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $instanceId = $this->getInstanceId();
+        $projectId = $this->getInstanceId();
```

Comment:
I think we should stick to the old name of "instance ID" nonetheless.
```suggestion
        $instanceId = $this->getInstanceId();
```